### PR TITLE
Correct link, for real this time

### PR DESCRIPTION
**Pull request by marcesher**

### CODE REVIEW

**Comment by willbarton** [comment]
oh, you

File: _posts/2015-12-10-readme-refresh-day.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Collaborators on software projects are frequently encouraged to write good insta
 ## Organizing the event
 
 A small group of existing CFPB developers organized the event, including planning and event-day activities,
-documented at [https://github.com/cfpb/readme-refresh-day](https://github.com/cfpb/readme-refresh-day).
+documented at [https://github.com/cfpb/readme-refresh-day](https://github.com/cfpb/readme_refresh_day).
```
Comment (Scotchester) [inline _posts/2015-12-10-readme-refresh-day.md]: Wanna correct the text of the link, too? ;)

**Comment by marcesher** [comment]
It's Friday, man.

File: _posts/2015-12-10-readme-refresh-day.md (hunk 1 above)
Comment (marcesher) [inline _posts/2015-12-10-readme-refresh-day.md]: I was going to but didn't want to deny you the opportunity of calling me out on it. 

One must follow protocol, after all.

**Pull request merged**